cmd/virtual-worker/cmd/root.go

```diff
@@ -0,0 +1,183 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/packethost/pkg/log"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+	"github.com/tinkerbell/tink/client"
+	tinkWorker "github.com/tinkerbell/tink/cmd/tink-worker/worker"
+	"github.com/tinkerbell/tink/cmd/virtual-worker/worker"
+	pb "github.com/tinkerbell/tink/protos/workflow"
+	"google.golang.org/grpc"
+)
+
+const (
+	defaultRetryIntervalSeconds       = 3
+	defaultRetryCount                 = 3
+	defaultMaxFileSize          int64 = 10 * 1024 * 1024 // 10MB
```

Comment:
you can get rid of the `int64` here and leave it a pure/untyped constant for longer and let the compiler figure out the type. Same as tink-worker does.

Reply:
done